modules/dev-tools/src/configuration/get-esbuild-config.js

```diff
@@ -86,38 +66,51 @@ function umdWrapper(libName) {
   };
 }
 
+/**
+ *
+ * @param {String} opts.input - path to entry point
+ * @param {String} opts.output - output file path
+ */
+export async function getCJSExportConfig(opts) {
+  return {
+    entryPoints: [opts.input],
+    outfile: opts.output,
+    bundle: true,
+    format: 'cjs',
+    target: 'node16',
```

Comment:
Node 16 is no longer supported, but I guess a grace period could make sense. Maybe add a comment above this line?